package.json

```diff
@@ -45,6 +45,7 @@
     "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
+    "@openmrs/esm-module-config": "file:../openmrs-esm-module-config",
```

Comment:
We should not push code that installs from file. If you intend for this PR to not be mergeable, please [mark it in Github as a draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) so that I know that this isn't mergeable.

Reply:
Also I'm thinking this should actually be a `dependency`, not a `devDependency`? Since it's an in-browser dependency and not a build-time dependency.